apps/family-private-law/identity/aat.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+  name: fprl
+  namespace: family-private-law
+spec:
+  resourceID: "/subscriptions/1c4f0704-a29e-403d-b719-b90c34ef14c9/resourcegroups/managed-identities-aat-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/fprl-aat-mi"
+  clientID: "73b1dbde-e66e-48d8-886e-439d9db3c14f"
```

Comment:
We don't want this file adding back @AlokDatta ?

Reply:
That's right. Not sure how it came back. I have removed this now.